components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
 
 Just add members to any of these default policies, as described later, to quickly get your basic users up and running.
 
-### Policies and Roles in Automate
+## Working with Projects
 
-<!-- I do not like saying "user interface" because technically that includes the command line, too. But I can live with it if you think that is the lesser of two evils. -->
-IAM v2 introduces partial policy management from within Chef Automate in your browser.
+IAM projects are collections of resources created in Chef Automate or ingested from external data providers, including ChefInfra and ChefInspec.
+Projects are used in a policy to reduce the scope of permissions of that policy to just the resources tagged with the given projects.
```

Comment:
lets not use the word _tagged_ here if possible, theres not really that notion from the users perspective i dont think